src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side1;
+    private int side2;
+
+    public RightTriangle(String color, int side1, int side2) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+    }
+
+    public double getArea() {
```

Comment:
Consider using a constant for the magic number 0.5 in the getArea() method to give it a meaningful name and make the code more readable.